qiskit_ibm_runtime/utils/embeddings.py

```diff
@@ -81,6 +86,54 @@ def coupling_map(self) -> CouplingMap:
         return self._coupling_map
 
 
+@dataclass(frozen=True)
+class _Row:
+    r"""
+    A class representing a dense row of the standard heavy hex planar embedding.
+
+    Args:
+        num_qubits: The number of qubits in the dense row.
+        x_offset: The integer x-offset of the row.
+        down_links: The positions of dense row, wrt the first qubit, at which to place qubits below.
```

Comment:
I know I suggested this format, so I recognize this is an annoying comment, but what about using `slice` instead of `_Row`?

Eg for 27 qubit devices:
```python
slice(3, 10, 4)
slice(0, 10)
slice(1, 10, 4)
slice(1, 11)
slice(3, 10, 4)
```

Feel free to respond with "quit wasting my time"

Reply:
No that's fine, it's just a CTRL+F, no big deal :P 
As long as you suggest a different name, because `slice` is something else already

Reply:
Actually, `range` instead of `slice`, and maybe just `Iterable[int]` instead of `range` while at it. Essentially, this becomes a sparse format for anything that can be specified row-wise.

Reply:
Oh, I thought you meant that you wanted a different name for the dataclass